src/controllers/proverControllers/proverController.ts

```diff
@@ -183,17 +184,92 @@ export const getVersion = async (req: any, res: any) => {
   }
 };
 
-//Generate Proof for the public and secret input
+// Initialize Redis client using environment variables
+const redisClient = new Redis({
+  host: process.env.REDIS_HOST,
+  port: parseInt(process.env.REDIS_PORT || "6379"),
+  password: process.env.REDIS_PASSWORD,
```

Comment:
Please add checks for the environment variables like the below:
https://github.com/marlinprotocol/Avail_server/blob/master/src/controllers/proverControllers/proverController.ts#L47-L54

Reply:
The steps, if needed and the env file contents must also be updated in the readme.